### ISSUE TITLE
Couldn't find preset "es2015-loose"

### ISSUE THREAD

**Issue by ruskid** [rethinkdb/horizon#798]
Fresh install of version 2. Still can't use horizon with React Native.

Error: Couldn't find preset "es2015-loose" relative to directory "/home/demin/startup/client/node_modules/@horizon/client"

> ```
> at /home/demin/startup/client/node_modules/babel-core/lib/transformation/file/options/option-manager.js:321:17
> ```

Server version: 2
Client version: 2

**Comment by deontologician** [comment]
This could be a babel version issue, do you have babel installed globally?

**Comment by lirbank** [comment]
@ruskid it doesn't work unless you remove the babel config, see this comment for details on getting it up and running https://github.com/rethinkdb/horizon/issues/255#issuecomment-241173730

**Comment by MikeShi42** [comment]
Yup just disable your babel config. Trying to get this at least noted in the docs in #750...

**Comment by jimthedev** [comment]
Unrelated to this issue but es2015-loose has been deprecated: https://github.com/bkonkle/babel-preset-es2015-loose
